homework-submission/week2hw/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import "./App.css";
+import todoList from "./data/todos.json";
+import Description from "./components/Description";
+
+class App extends Component {
+  state = {
+    todoList
+  };
+
+  clickHandler = e => {
+    let tempArr = this.state.todoList.slice();
+
+    console.log(e.target.id);
+
+    tempArr[e.target.id - 1].done = !tempArr[e.target.id - 1].done;
```

Comment:
I know how to do it with **map** but i think it's not necessary to use **map** since i mutate one item only
Also i would prefer to use forEach for mutating the original array